tests/tests/swfs/avm2/pixelbender_effect_glassDisplace/test.toml

```diff
@@ -2,7 +2,7 @@ num_frames = 1
 
 [image_comparison]
 tolerance = 1
-max_outliers = 34
+max_outliers = 1640
```

Comment:
This may not need to be this high, try just increasing the tolerance by 1 or 2 and use outliers for the really big outliers

Reply:
Bumping tolerance to 3 helped a little, but there's still a large number of outliers, unfortunately.